patches/minecraft/net/minecraft/world/entity/projectile/FireworkRocketEntity.java.patch

```diff
@@ -1,13 +1,19 @@
 --- a/net/minecraft/world/entity/projectile/FireworkRocketEntity.java
 +++ b/net/minecraft/world/entity/projectile/FireworkRocketEntity.java
-@@ -150,6 +_,13 @@
+@@ -150,6 +_,19 @@
  
     }
  
 +   @Override
 +   protected void m_6532_(HitResult result) {
-+      if (result.m_6662_() == HitResult.Type.MISS || !net.minecraftforge.event.ForgeEventFactory.onProjectileImpact(this, result)) {
-+         super.m_6532_(result);
++      if (result.m_6662_() == HitResult.Type.MISS) {
++         switch (net.minecraftforge.event.ForgeEventFactory.onProjectileImpact(this, result))
++         {
```

Comment:
Formatting: In Minecraft code, the brace style should follow the decompiler output, which is braces on the same line.